raiden/transfer/node.py

```diff
@@ -1148,7 +1148,13 @@ def is_transaction_effect_satisfied(
         and isinstance(transaction, ContractSendChannelClose)
         and state_change.token_network_identifier == transaction.token_network_identifier
         and state_change.channel_identifier == transaction.channel_identifier
+    ) or (
```

Comment:
why not place an implicit `or` on line `1147`?

```python
isinstance(state_change, (ContractReceiveChannelClosed, ContractReceiveChannelClosedLight))
and isinstance(transaction, ContractSendChannelClose)
and state_change.token_network_identifier == transaction.token_network_identifier
and state_change.channel_identifier == transaction.channel_identifier
```

Reply:
You are right

Reply:
feel free to push the change or leave as-is, this isn't a blocker for me